tunedev/Banka
# #165396225 database setup

#### What does this PR do?
This pr integrates database using Postgresql

#### Description of Task to be completed?
- Replace all file structures with the database
- Update test cases to accommodate changes

#### How should this be manually tested?
- Clone this repo and cd into the root folder
- Then check out to branch ch-database-setup-165396225
- Inspect the database directory to properly set-up your .env file
- start the app by running command "npm run devstart"
- Test all endpoints that were previously working

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165396225](https://www.pivotaltracker.com/story/show/165396225)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
